website/client/router.js

```diff
@@ -26,7 +29,15 @@ export default new VueRouter({
       component: ParentPage,
       children: [
         {path: 'tavern', component: Page},
-        {path: 'inbox', component: Page},
+        {
+          path: 'inbox',
+          component: InboxPage,
+        },
+        {
```

Comment:
I'm pretty sure this should be a child route, but the child routes seem to keep the html of the parent. Still looking into this.